packages/flutter_solidart/lib/src/widgets/solid.dart

```diff
@@ -272,21 +276,21 @@ class Solid extends StatefulWidget {
         ),
         _canAutoDisposeProviders = autoDispose;
 
-  /// Provide a single or multiple [SolidElement]s to a new route.
+  /// Provide a single or multiple [ProviderElement]s to a new route.
   ///
   /// This is useful for passing multiple providers to modals, because are
   /// spawned in a new tree.
-  factory Solid.value({
+  factory ProviderScope.value({
```

Comment:
ProviderScope.value is not a bad name. It's similar to `Provider.value`, the `provider` package.

Reply:
I didn't know this. Let's keep it then.